### ISSUE TITLE
In Browsershot::createScreenshotCommand the url is not escaped

### ISSUE THREAD

**Issue by tommyseus** [spatie/browsershot#67]
I think in Browsershot::createScreenshotCommand the url is not escaped.
Is it possible to add some quotes or using escapeshellarg()?

```
Browsershot::url('https://www.google.de/search?num=50&site=&source=hp&q=test&oq=test')
    ->timeout(60)
    ->windowSize(1280, 1280)
    ->save(__DIR__ . '/dummy.png');
```

Version: 2.0.0

**Comment by freekmurze** [comment]
Feel free to PR a fix for this.

**Comment by freekmurze** [comment]
Fixed in `v2.0.1`
